src/java.base/share/classes/java/lang/StackTraceElement.java

```diff
@@ -355,27 +354,46 @@ public boolean isNativeMethod() {
      * @revised 9
      * @see    Throwable#printStackTrace()
      */
+    @Override
     public String toString() {
-        String s = "";
-        if (!dropClassLoaderName() && classLoaderName != null &&
-                !classLoaderName.isEmpty()) {
-            s += classLoaderName + "/";
+        int estimatedLength = Objects.requireNonNullElse(classLoaderName, "").length() + 1
```

Comment:
Nit: suggest to refactor this and define a private static `length(String s)` method.

Reply:
updated